pkg/apis/provisioning/v1alpha5/provisioner_validation.go

```diff
@@ -29,7 +29,6 @@ import (
 
 var (
 	SupportedNodeSelectorOps sets.String = sets.NewString(string(v1.NodeSelectorOpIn), string(v1.NodeSelectorOpNotIn), string(v1.NodeSelectorOpExists), string(v1.NodeSelectorOpDoesNotExist))
-	SupportedProvisionerOps  sets.String = sets.NewString(string(v1.NodeSelectorOpIn), string(v1.NodeSelectorOpExists))
```

Comment:
Semantically, what does `DoesNotExist` mean at the provisioner level?

Reply:
I am thinking using it later for provisioning matching when we allow arbitrary requirement labels. So a pod can specify a label and the provisioner will use DoesNotExist to exclude itself from matching to these pods.

Reply:
Let's take this out an include it when we get there. Right now, a provisioner won't match a pod if the custom label isn't defined (by design), so the exists operator is a noop.

Reply:
done